### PR TITLE
Implement eden ls command, fix style

### PR DESCRIPTION
**Pull request by prog893**
- fix json import
- implement eden ls (print existing environments)
- rename remote-remove to remote-rm

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,7 +193,7 @@ Successfully pushed profile default to DynamoDB table eden
 
 # use remote-delete to remove remote profiles
```
Comment (toVersus) [inline README.md]:
```suggestion
# use remote-rm to remove remote profiles
```

Comment (toVersus) [reply README.md]: In addition, change `remote-remove` to `remote-rm`.

File: aws_eden_cli/cmdline.py [hunk 2]
```diff
@@ -86,6 +90,25 @@ def create_parser():
...
+    environments = dynamodb.fetch_all_environments(table)
+
+    for profile_name in environments:
+        logger.info(f"Profile {profile_name}:")
+
+        for environment in environments[profile_name]:
+            last_updated = datetime.datetime.fromtimestamp(float(environment['last_updated_time']))
+            logger.info(f"{environment['name']} {environment['endpoint']} (last updated: {last_updated.isoformat()})")
+
+        logger.info("")
```
Comment (toVersus) [inline aws_eden_cli/cmdline.py:107]: Is that necessary for a new line?

Comment (prog893) [reply aws_eden_cli/cmdline.py:107]: eden prints environments in sections, by profile used for creation. I think it is better to separate those sections with a newline, and I do not think it is really necessary to make sure there is no newline in the end of section.

**Review by toVersus — APPROVED** [review]
LGTM

**Pull request merged**